src/main/resources/static/service_openapi.yaml

```diff
@@ -550,6 +550,17 @@ components:
           description: |
             The URL of the Workspace Manager instance which manages this workspace. This must be
             in Janitor's configured allowlist.
+    AzureManagedIdentity:
```

Comment:
do we want to specify `userManaged` identity? just to differentiate it from system assigned ones...or at this level, it could be both?

Reply:
The System-assigned managed identities are deleted with resources they've been created on. So, there is no need to handle system-assigned MI's here. As for the naming, I think AzureManagedIdentity is good enough.